.travis.yml

```diff
@@ -2,28 +2,32 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
+php: [5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
 
 matrix:
   fast_finish: true
   include:
     - php: 5.3
+      dist: precise
+    - php: 7.1
       env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
+    - php: 7.1
       env: DEPENDENCIES=dev
 
 cache:
   directories:
     - $HOME/.composer/cache/files
+    - $HOME/symfony-bridge/.phpunit
 
 before_install:
-  - composer self-update
-  - if [ "$DEPENDENCIES" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi;
+  - if [ "$DEPENDENCIES" = "dev" ]; then composer config minimum-stability dev; fi;
 
 install:
   - composer update $COMPOSER_FLAGS
+  - vendor/bin/simple-phpunit install
 
-script: phpunit -v --coverage-clover=coverage.clover
+script: vendor/bin/simple-phpunit -v --coverage-clover=coverage.clover
```

Comment:
What is `simple-phpunit` and how it's better than regular phpunit?

Reply:
https://symfony.com/doc/current/components/phpunit_bridge.html

Here it is basically used to run tests both on PHP 5 and 7 (not easy with PHPUnit directly).

Reply:
for you @dunglas

Reply:
Never had any issues with this. Just use PHPUnit version that Travis CI gives you (the code you've changed was doing that) and PHPUnit version compatible with used PHP version would be picked automatically.

Reply:
@aik099 Travis CI gives you a version of PHPUnit compatible with the PHP runtime. but when it gives you PHPUnit 4.8, it might give you a version older than 4.8.35, and so without the new namespaced class names for TestCase, making hard to have a testsuite compatible with PHPUnit 6 as well (which is used by default on Travis for PHP 7.2 for instance)

Reply:
using a phpunit version installed through composer could also be working. But the drawback is that it would install an old unmaintained version of phpunit when using `--prefer-lowest`. And this does not make sense (we want to test the lowest versions of our deps, but we can still use an uptodate testing tool for that, avoiding to be hit by old PHPUnit bugs)